### PR TITLE
Initial LoRaWAN

### PR DESCRIPTION
**Pull request by lthiery**

### CODE REVIEW

File: docs/lorawan-on-helium/lorawan-on-helium.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+```
+
+From the gateways perspective, the (AppEUI, DevEUI) may be used to figure out if
+one or more OUI's are interested in the Join Request.
+
+This is done by querying every filter maintained and written to the blockchain
+by every OUI. The filters themselves are _xorhash_ tables, thus the AppEui and
+DevEui cannot be extracted. Only with the AppEui and DevEui in-hand can the
+gateway can run it through the table and see if it is in the table.
```
Comment (kent-williams) [inline docs/lorawan-on-helium/lorawan-on-helium.mdx]: Extra `can`

File: docs/lorawan-on-helium/frequency-plans/frequency-plans.mdx [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+id: frequency-plans
+hide_title: true
+sidebar_label: Frequency Plans
+slug: /lorawan-on-helium/frequency-plans
+---
+
+## Frequencies on the Helium Network
+
+Helium's LoRaWAN Network is built upon 8-channel gateways. This page will help
+you confirm the appropriate frequencies for gateway configuration each country.
```
Comment (kent-williams) [inline docs/lorawan-on-helium/frequency-plans/frequency-plans.mdx]: Needs an extra `for` after configuration.

**Comment by kent-williams** [comment]
Left two small comments. Nice work, LGTM!

**Review by kent-williams — DISMISSED** [review]

File: docs/lorawan-on-helium/frequency-plans/frequency-plans.mdx (hunk 2 above)
Comment (lthiery) [reply docs/lorawan-on-helium/frequency-plans/frequency-plans.mdx]: Good catch, thanks.

File: docs/lorawan-on-helium/lorawan-on-helium.mdx (hunk 1 above)
Comment (lthiery) [reply docs/lorawan-on-helium/lorawan-on-helium.mdx]: Thank you

**Review by kent-williams — APPROVED** [review]

**Pull request merged**